### ISSUE TITLE
Exception when delivering videos because of missing token

### ISSUE THREAD

**Issue by migster42** [sulu/sulu#6880]
| Q | A
| --- | ---
| Sulu Version | 2.4.6
| PHP Version | 8.1.10

# Actual Behavior

After upgrading from 2.4.5 to 2.4.6 videos (mp4 files) are not delivered.

```
Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException:
The token storage contains no authentication token. One possible reason may be that there is no firewall configured for this URL.

  at vendor/symfony/security-core/Authorization/AuthorizationChecker.php:74
  at Symfony\Component\Security\Core\Authorization\AuthorizationChecker->isGranted('view', object(SecurityCondition))
     (vendor/sulu/sulu/src/Sulu/Component/Security/Authorization/SecurityChecker.php:51)
  at Sulu\Component\Security\Authorization\SecurityChecker->hasPermission(object(SecurityCondition), 'view')
     (vendor/sulu/sulu/src/Sulu/Component/Security/Authorization/AbstractSecurityChecker.php:25)
  at Sulu\Component\Security\Authorization\AbstractSecurityChecker->checkPermission(object(SecurityCondition), 'view')
     (vendor/sulu/sulu/src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php:153)
  at Sulu\Bundle\MediaBundle\Controller\MediaStreamController->downloadAction(object(Request), '3')
     (vendor/symfony/http-kernel/HttpKernel.php:163)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:75)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:57)     
```

# Expected Behavior

Videos are delivered without exception.

# Steps to Reproduce

Upload a video on admin ui then logout and call the url of the `MediaStreamController`, e.g. `/media/3/download/video.mp4?v=1`.

# Possible Solutions

I think the issue is because of this change: https://github.com/sulu/sulu/commit/711053c0cc6601947014eace41ebbf0b685f98b6#diff-7f0014f16d2c2e13909d32c68a85006c7eb68d7f6fb622fcc26c27cb24c6515cL42

If I add the token check again it works fine.

**Comment by alexander-schranz** [comment]
@migster42 can you also tell use which Symfony Version you are using and if you are using the new or old authenticator system of symfony. New one has this line old: https://github.com/sulu/skeleton/blob/6384df0905915a593419caa6cbae5dff79265cd0/config/packages/security.yaml#L2

**Comment by migster42** [comment]
@alexander-schranz I use Symfony 5.4.15 and my `security.yaml` is the same as in the skeleton of sulu version 2.4.6: https://github.com/sulu/skeleton/blob/2.4.6/config/packages/security.yaml

With `enable_authenticator_manager: true` it works for me. Thank you!

**Comment by alexander-schranz** [comment]
@migster42 we did fix the issue in #6881 so it should also work for the old authenticator system.
